client/components/ScenariosList/index.jsx

```diff
@@ -43,6 +43,20 @@ const ScenarioEntries = ({ scenarioData, isLoggedIn }) => {
                         </Button>
                     </Card.Content>
                 )}
+                {isLoggedIn && (
+                    <Card.Content extra>
+                        <Button
+                            basic
+                            fluid
+                            color="black"
+                            push="true"
+                            as={Link}
+                            to={{ pathname: `/run/${id}` }}
```

Comment:
Likewise here, I think we can consolidate the `/moment/:id` route and the `/run/:id` route.

Reply:
Perhaps we don't even want the title of the scenario listing to be linked and only give users the option to explicitly run the scenario instead of accidentally clicking into it.